client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -67,13 +69,22 @@ const LowerJaw = ({
   }, [challengeHasErrors, hint]);
 
   useEffect(() => {
-    if (challengeIsCompleted && submitButtonRef?.current) {
+    if (challengeHasBeenCompleted && submitButtonRef?.current) {
       submitButtonRef.current.focus();
       setTimeout(() => {
         setTestBtnariaHidden(true);
       }, 500);
     }
 
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [challengeHasBeenCompleted]);
+
+  useEffect(() => {
+    if (!challengeHasBeenCompleted) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      setChallengeHasBeenCompleted(challengeIsCompleted!);
```

Comment:
```suggestion
      setChallengeHasBeenCompleted(!!challengeIsCompleted);
```
Rather than disabling the linter, can we convert it?

Reply:
good catch. added a check if `challengeIsCompleted` is truthy before setting `challengeHasBeenCompleted`